pkg/build/controller/buildpod/factory.go

```diff
@@ -0,0 +1,213 @@
+package buildpod
```

Comment:
> It is common in go to have methods of a single struct span multiple files.

and maybe someday someone will convince me that's a benefit :)
(showing my java predilection, i know)